server/memphis_handlers_producers.go

```diff
@@ -186,6 +186,20 @@ func (s *Server) createProducerDirectCommon(c *client, pName, pType, pConnection
 			serv.Errorf("createProducerDirectCommon: Producer " + pName + " at station " + pStationName.external + ": " + err.Error())
 		}
 
+		update := models.ConfigurationsUpdate{
```

Comment:
let's talk about it, the face the dls is enabled not means you have to send notificatino